service/openapi/src/main/java/org/dotwebstack/framework/service/openapi/response/ResponseWriteContext.java

```diff
@@ -22,6 +22,10 @@
 
   URI uri;
 
+  public boolean hasSchema() {
+    return getResponseObject().getSummary() != null;
```

Comment:
In DWS worden voor == en != met objecten altijd de 'Objects' hulpmethodes gebruikt:
```suggestion
    return Objects.nonNull(getResponseObject().getSummary());
```